.github/workflows/linux-workflow.yml

```diff
@@ -66,14 +60,14 @@ jobs:
             appimage: false
             experimental: false
           - os: ubuntu-18.04
-            platform: x86
+            platform: x64
             compiler: gcc
             cmakeflags: -DCMAKE_DISABLE_PRECOMPILE_HEADERS=ON
             detail: " nopch"
             appimage: false
             experimental: false
           - os: ubuntu-18.04
-            platform: x86
+            platform: x64
```

Comment:
`platform` should be moved out of include so it doesn't need to be set for each job

Reply:
unfortunately this is not how this works it seems, i tried it and it ends up not creating all the necessary builds.  Each include object seems to have to completely describe the run configuration -- and docs don't suggest otherwise https://docs.github.com/en/actions/learn-github-actions/workflow-syntax-for-github-actions#jobsjob_idstrategymatrix